code/modules/reagents/chemistry/recipes/others.dm

```diff
@@ -528,6 +528,16 @@
 /datum/chemical_reaction/life/on_reaction(datum/reagents/holder, created_volume)
 	chemical_mob_spawn(holder, rand(1, round(created_volume, 1)), "Life") // Lol.
 
+//This is missing, I'm adding it back (see tgwiki). Not sure why we don't have it.
+/datum/chemical_reaction/life_friendly
```

Comment:
Atomize this port away.

Reply:
Fair.

Reply:
for this and a lot of the other stuff I'm fine with it not being atomized this time but please do it if you can it does speed up review because we won't have to go through stuff unlrelated to the PR or only slightly related. the more stuff is in a pr the longer it takes to pass it in most cases.

Reply:
Yeah, don't worry, next time I'll atomized the heck out of everything.

Reply:
The _hecc_ Kevin.